cli/src/checks.rs

```diff
@@ -192,7 +192,7 @@ mod tests {
             },
             value: json!(account_balance),
         });
-        let pubkey = solana_sdk::pubkey::new_rand();
```

Comment:
I wonder why we explicitly use solana_sdk/solana_pubkey here instead of importing them once

Reply:
yeah idk, it happpens a lot in the whole repo

Reply:
maybe it is ok to use `solana_pubkey::new_rand()` because new_rand is not descriptive enough